WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -533,10 +537,7 @@ targetted_decisions = {
 						NOT = { religion_group = ROOT }
 					}
 				}
-				random = {
-					chance = 60
-					add_trait = sympathy_christendom
-				}
+				add_trait = sympathy_christendom
```

Comment:
It makes no sense that it is. If I were once an Atheist and got Saved and became a Christian, I would harbor sympathy and understanding for my former belief system. I would be less harsh or disagreeing with them and would want them to join me in the faith. You don't suddenly hate your old religion because you forsake a deity you now deem false.

For example, Ex-Muslims may hate Islam, but they don't inherently hate other Muslims, and in fact, want to help them leve the faith, too.